docs/guides/instances.mdx

```diff
@@ -157,8 +155,8 @@ sampler2 = SamplerV2(mode=backend2)
 If you do not specify an instance, the code will select one in the following order:
 
 1. If your account only has access to one instance, it is selected by default.
-2. If your account has access to multiple instances but only one can access the requested QPU, the instance with access is selected.
-3. In all other cases, the code selects the first instance other than the Open Plan instance that has access to the quantum computer.
+2. If your account has access to multiple instances but only one can access the requested QPU, the instance with access is selected. If this instance is not associated with a free plan, a cost will be incurred, except under special circumstances, such as if you have credits that give you free usage.
```

Comment:
can you maybe clarify with Marco or Val if this works like this? and if naming "credits" is correct

Reply:
I was referring that further clarification was needed for the last part of the sentence "If this instance is not associated with a free plan, a cost will be incurred, except under special circumstances, such as if you have credits that give you free usage."

I don't think there are special circumstances, for non-free plans the time is always billed and just if you happen to have credits it will subtract from there, but there is no free usage on non-free plans